docs/_data/features.ts

```diff
@@ -55,12 +56,21 @@ export const mail: FeatureListItem[] = [
 	{ text: "Optional Rainloop installation" },
 ];
 
+export const dns: FeatureListItem[] = [
+	{ text: "Create your own nameservers" },
+	{ text: "Easy setup for a DNS cluster" },
```

Comment:
```suggestion
	{ text: "Easy DNS cluster setup" },
```

Sounds a bit better to me